### ISSUE TITLE
POW DNS routing breaks Docker

### ISSUE THREAD

**Issue by dougvk** [basecamp/pow#528]
All my Docker containers couldn't connect to the internet. It took a while to figure out it was POW.

**Comment by JustinAiken** [comment]
:+1: 

Conversely, installing the mac docker beta seems to have broken pow :/

**Comment by justincormack** [comment]
Yes, we have had a lot of reports of issues with `pow`. If anyone needs a fast track invite to help fix this let me know.

**Comment by JustinAiken** [comment]
Ooh, I was wrong - docker didn't break pow, that was my VPN.

But pow definitely breaks docker :stuck_out_tongue:

**Comment by ktheory** [comment]
I have an inverse issue, which I'm presuming is related. After starting or stopping Docker for Mac Beta Version 1.11.0-beta6 (build: 5404), Pow port forwarding doesn't work.

I.e., running `curl -vH host:pow "127.0.0.1/status.json"` fails with "Connection refused" after starting or stopping Docker.

I found a workaround (from the Troubleshooting guide) to reload packet filter rules fixes Pow, without breaking docker:

```
sudo pfctl -f /etc/pf.conf; sudo pfctl -e
```

**Comment by ktheory** [comment]
FYI, I tried using the default packet filter namespace from #521, and that didn't fix broken Pow rules after starting/stopping docker. My `pfctl`-fu is too weak to debug further.

Also, seems fair to link to the corresponding [Docker for Mac forum discussion about Pow](https://forums.docker.com/t/dns-issue-with-building-ruby-image/8060/4).

Both Pow & Docker modify OS X network settings; and it has not (yet) been established what app change could mitigate/resolve the conflict.

**Comment by stereodenis** [comment]
any updates on this?

**Comment by ktheory** [comment]
@stereodenis WFM w/ recent Docker for Mac betas: https://forums.docker.com/t/dns-issue-with-building-ruby-image/8060/9

**Comment by bobmaerten** [comment]
I think this is not directly related to Docker, as I use Veertu for virtualization, which uses native `hypervisor.framework` from OS X, and I have similar problems with Pow. 
As Docker relies on osx native virtualization, I really think the problem is elsewhere.

**Comment by Yurmusp** [comment]
I have similar problem on Sierra.
It seems it happens when I interrupt program that runs as docker by ^C.
It seems docker does not close something correctly in this case and pow breaks.
None of recipes here worked for me to fix it.
Thing that works for me was to unload and load pow again with these commands:
    launchctl unload ~/Library/LaunchAgents/cx.pow.powd.plist
    launchctl load     ~/Library/LaunchAgents/cx.pow.powd.plist
